unity/EditorPlugin/PluginEntryPoint.cs

```diff
@@ -1,85 +1,83 @@
 using System;
-using System.Collections.Generic;
 using System.Diagnostics;
 using System.IO;
-using System.Linq;
 using System.Reflection;
-using JetBrains.Annotations;
-using JetBrains.Collections.Viewable;
 using JetBrains.Core;
 using JetBrains.Rider.Model.Unity.BackendUnity;
 using JetBrains.Diagnostics;
 using JetBrains.Lifetimes;
-using JetBrains.Rd;
-using JetBrains.Rd.Base;
-using JetBrains.Rd.Impl;
 using JetBrains.Rd.Tasks;
-using JetBrains.Rider.Model.Unity;
-using JetBrains.Rider.Unity.Editor.Logger;
-using JetBrains.Rider.Unity.Editor.NonUnity;
-using JetBrains.Rider.Unity.Editor.Utils;
 using UnityEditor;
 using UnityEditor.Callbacks;
 using Application = UnityEngine.Application;
 using Debug = UnityEngine.Debug;
 
+// DO NOT CHANGE NAMESPACE OR CLASSNAME!
```

Comment:
Do you propose, that we change the calls in the package to point this new namespace?

Reply:
No, the namespace for `EntryPoint` hasn't changed, and I'm not going to change it. It'll stay as it was and the existing reflection call will work as before. But I'll extend this comment to state that it's called from the package